# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libedit/hotfix-chartype.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libedit-20240517-3.1/src/chartype.h.vanilla	2024-08-02 16:44:16.087015612 +0200
+++ libedit-20240517-3.1/src/chartype.h	2024-08-02 16:44:48.477015714 +0200
@@ -44,7 +44,7 @@
 /* In many places it is assumed that the first 127 code points are ASCII
  * compatible, so ensure wchar_t indeed does ISO 10646 and not some other
  * funky encoding that could break us in weird and wonderful ways. */
-	#error wchar_t must store ISO 10646 characters
+	//#error wchar_t must store ISO 10646 characters
 #endif
 #endif
 
